calico.go

```diff
@@ -306,6 +307,33 @@ func cmdDel(args *skel.CmdArgs) error {
 		"Node":         nodename,
 	}).Info("Extracted identifiers")
 
+	calicoClient, err := CreateClient(conf)
+	if err != nil {
+		return err
+	}
+
+	ep := api.WorkloadEndpointMetadata{
+		Name:             args.IfName,
+		Node:             nodename,
+		ActiveInstanceID: args.ContainerID,
+		Orchestrator:     orchestrator,
+		Workload:         workload,
+	}
+
+	wep, err := calicoClient.WorkloadEndpoints().Get(ep)
```

Comment:
ActiveInstanceID is part of the Metadata, but not the key, so other 4 fields that make up the key are still unchanged